From 451f21e36dac14479287d47790e976adea002c13 Mon Sep 17 00:00:00 2001
From: Henrik Grimler <henrik@grimler.se>
Date: Mon, 20 Dec 2021 11:51:05 +0100
Subject: [PATCH 13/16] dt-bindings: arm: samsung: document Klimt WiFi board
 binding

Add binding for Galaxy Tab S 8.4", based on Exynos 5420 with codename
klimt-wifi. The tablet has similar hardware to Chagall WiFi, but a
smaller battery, smaller screen and another touchscreen controller.

Signed-off-by: Henrik Grimler <henrik@grimler.se>
---
 .../devicetree/bindings/arm/samsung/samsung-boards.yaml          | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/samsung/samsung-boards.yaml b/Documentation/devicetree/bindings/arm/samsung/samsung-boards.yaml
index c385e6bfaed1..abac01def63c 100644
--- a/Documentation/devicetree/bindings/arm/samsung/samsung-boards.yaml
+++ b/Documentation/devicetree/bindings/arm/samsung/samsung-boards.yaml
@@ -141,6 +141,7 @@ properties:
           - enum:
               - insignal,arndale-octa           # Insignal Arndale Octa
               - samsung,chagall-wifi            # Samsung SM-T800
+              - samsung,klimt-wifi              # Samsung SM-T700
               - samsung,smdk5420                # Samsung SMDK5420 eval
           - const: samsung,exynos5420
           - const: samsung,exynos5
-- 
2.34.1

